services/autoscaling/src/simcore_service_autoscaling/utils/utils_docker.py

```diff
@@ -330,12 +330,14 @@ async def get_docker_swarm_join_bash_command() -> str:
 def get_docker_login_on_start_bash_command(registry_settings: RegistrySettings) -> str:
     return " ".join(
         [
+            "echo",
+            f'"{registry_settings.REGISTRY_PW.get_secret_value()}"',
+            "|",
             "docker",
             "login",
             "--username",
             registry_settings.REGISTRY_USER,
-            "--password",
-            registry_settings.REGISTRY_PW.get_secret_value(),
+            "--password-stdin",
```

Comment:
I guess before this PR login into custom docker registries would not work? Maybe add a quick note in the PR description for posterity if this is so :)

Reply:
no it was already doing it since #3876 . This is just using the more "secure" way.